docs/source/api.rst

```diff
@@ -108,3 +108,11 @@ Multi-Category Metrics
     Contingency.heidke_score
     Contingency.peirce_score
     Contingency.gerrity_score
+
+Tests
```

Comment:
Make sure to call this "Comparative" or "Comparisons" since you've changed the module name. Same for `quick-start`.